### DISCUSSION TITLE
how to break long flow to new line?

### DISCUSSION THREAD

**Discussion by nonkr** [mermaid-js/mermaid#4306] [Q&A]
If the flow step is very long in one direction, how to make it break it into multiple lines.
like
![IMG_9552](https://user-images.githubusercontent.com/1668287/232449394-8aeb3741-d1e7-42f0-b000-3e33cc25e542.jpeg)
to
![IMG_9553](https://user-images.githubusercontent.com/1668287/232450188-ea6b9ee9-e50b-4764-8005-e7c61eae7a65.jpeg)

**Comment by elzup** [comment]
layout, (dimension, column) is not supported
Many issues have been posted

Alternative
split steps and use subflow

```mermaid
flowchart LR
	direction LR
	g1-->g2
	subgraph g1
		direction TB
		a1-->a2
    end
	subgraph g2
	    a3-->a4
    end
```
